api/v01/api.rb

```diff
@@ -27,6 +27,43 @@ class Api < Grape::API
       use HttpAcceptLanguage::Middleware
 
       helpers do
+        def redis_count
+          OptimizerWrapper.config[:redis_count]
+        end
+
+        def count_time
+          @count_time ||= Time.now.utc
+        end
+
+        def count_key(operation)
+          @count_key ||= [
+            [:optimizer, operation, count_time.to_s[0..9]],
+            [:key, params[:api_key]],
+            [:ip, request.ip],
```

Comment:
this commit is missing: https://github.com/Mapotempo/geocoder-api/commit/0b2eafe588399b30f976064bb48704d26d9e84e1